nixnet/database/_frame.py

```diff
@@ -83,7 +86,8 @@ def dbc_attributes(self):
         # type: () -> _dbc_attributes.DbcAttributeCollection
         """:any:`nixnet.database._dbc_attributes.DbcAttributeCollection`: Access the frame's DBC attributes."""
         if self._dbc_attributes is None:
-            self._dbc_attributes = _dbc_attributes.DbcAttributeCollection(self._handle)
+            self._dbc_attributes = _dbc_attributes.DbcAttributeCollection(
```

Comment:
Why is this commit including an unrelated change?

Reply:
I figured I would fix the pep8 violation (over the max line length) while I was in the file. I just looked at tox.ini and saw it's set to 120 characters max, so I'll change this back.

Reply:
Our official guideline is to aim for below 80 characters but only enforce below 120.

... we still need to get that guideline published publicly.